pages/2020/schedule.rst

```diff
@@ -0,0 +1,258 @@
+.. title: PyAstro 2020 schedule
+
+.. table::
+  :widths: 10, 100
+
+  +--------+--------------------------------------------------------------------------------+
+  |        |Monday                                                                          |
+  +========+================================================================================+
+  |9:00am  |Coffee                                                                          |
+  +--------+--------------------------------------------------------------------------------+
+  |9:30am  |*Kelle Cruz* - **Welcome talk**                                                 |
```

Comment:
:laughing:

Reply:
Yeah we should probably confirm this with her first :P

Reply:
There's no link to the schedule from the main page as of yet so this isn't really "public"

Reply:
Removed